### PR TITLE
Typo Fixes-3

### LINKED ISSUE #590: The `Fine-tuning a pretrained model` notebook doesn't install the `evaluate` library.
The [Fine-tuning a pretrained model](https://github.com/huggingface/notebooks/blob/main/transformers_doc/en/training.ipynb) notebook that can be found here, https://github.com/huggingface/transformers/tree/main/notebooks, has a minor inconvenience.

Image

It happens because we didn't install the `evaluate` library.

Image

We should add `evaluate` to the import list.

To make the experience smooth, we should install `evaluate` at the top of the notebook.

### PR DESCRIPTION
**Pull request by ParagEkbote**
# What does this PR do?
This PR aims to fix minor typos present in the notebooks. Fixes #590, #534, #519

## Who can review?

@stevhliu

### CODE REVIEW

**Review by pcuenca — COMMENTED** [review]
I'm fine with the changes in `semantic_segmentation` and `training`, but I don't think it's necessary to rename the variable that represents the metric in `summarization`. The original issue report was about using `evaluate.load`, but I don't think it's necessary as `load` is imported in the same cell.

**Comment by ParagEkbote** [comment]
I have reverted the changes. Could you please review?

cc: @pcuenca

**Comment by ParagEkbote** [comment]
Could you please review the changes?

cc: @pcuenca

**Comment by pcuenca** [comment]
Hello @ParagEkbote. Sorry for the delay, but there's no need to ping multiple times.

**Review by pcuenca — APPROVED** [review]
